lib/teams/emr-on-eks-team.ts

```diff
@@ -47,9 +47,13 @@ export interface EmrEksTeamProps extends TeamProps {
   */
   virtualClusterName: string,
   /**
-   * List of execution role to associated with the VC namespace
+   * List of execution role to associated with the VC namespace {@link ExecutionRoleDefinition}
    */
-  executionRoles: ExecutionRoleDefinition[] 
+  executionRoles: ExecutionRoleDefinition[]
```

Comment:
minor: semicolon after each member of class and interface. Please refer to these code conventions: https://google.github.io/styleguide/tsguide.html#member-property-declarations